components/swatchgroup/stories/swatchgroup.stories.js

```diff
@@ -25,7 +19,7 @@ import { RoundingTemplate, Template } from "./template.js";
  *
  * ### Density
  *
- * Swatch groups come in 3 densities: regular (default), compact, and spacious. Compact and spacious densities retain the same swatch size as regular density, but have less or more padding between each swatch, respectively.
+ * Swatch groups come in several densities: regular (default), spacious, medium and large. Each density retains the same swatch size, but have less or more padding between each swatch, respectively.
```

Comment:
Interesting that the densities include spacious, medium, and large. What's the distinction between spacious vs. large? Do they set different properties?

Reply:
I think I'd like to see T-shirt sizing be distinct from densities since we already have a convention for size="l" for example.